### ISSUE TITLE
Can't run spectral 4.1.0 when installed via npm

### ISSUE THREAD

**Issue by johlo** [stoplightio/spectral#524]
**Describe the bug**
I installed spectral 4.1.0 via npm, when running the `spectral` command it errors out.
When I install 4.0.3 via npm the command works. 

**To Reproduce**

1. `$ npm install -g @stoplight/spectral`
2. `$ spectral`
    This gives an error:
```
internal/modules/cjs/loader.js:775
    throw err;
    ^

Error: Cannot find module 'tslib'
Require stack:
- /usr/local/lib/node_modules/@stoplight/spectral/dist/cli/commands/lint.js
- /usr/local/lib/node_modules/@stoplight/spectral/dist/cli/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:772:15)
    at Function.Module._load (internal/modules/cjs/loader.js:677:27)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@stoplight/spectral/dist/cli/commands/lint.js:3:17)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Module.require (internal/modules/cjs/loader.js:830:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/@stoplight/spectral/dist/cli/commands/lint.js',
    '/usr/local/lib/node_modules/@stoplight/spectral/dist/cli/index.js'
  ]
}
```

**Expected behavior**
No error when running

**Environment (remove any that are not applicable):**
 - Library version: 4.1.0
 - OS: mac os 10.13.5
 - npm: 6.11.13

**Additional context**
FYI, I also tried to installed the binary by running the `curl` from your install instructions, but that doesn't work:
```
$ curl -L https://raw.githack.com/stoplightio/spectral/master/install.sh
404: Not Found
```

**Comment by P0lip** [comment]
@johlo 
I'm sorry to hear you are having troubles with latest Spectral 4.1.
I can confirm it run fine under Node 12, but 10 version is in fact broken, so a fix is certainly needed.
In the meantime, you can switch over to Node 12 if you have such a possibility.
Re the 404 - the file location has changed, I'll update the readme.
You can find compiled binaries over here https://github.com/stoplightio/spectral/releases/tag/v4.1.0.
My apologies for any inconvenience caused.

**Comment by P0lip** [comment]
@johlo 
4.1.1 containing the fix has been released.
Let me know if you face any troubles.

**Comment by johlo** [comment]
Works now, thanks!

**Comment by philsturgeon** [comment]
Also the installer script 404 was due to the docs not being published after we merged v4.1.0 changes into master. The docs have been updated now, and soon they'll be auto-publishing so we don't have to remember.
